"custom plots" tab is now more intuitive and consistent

JohnnyDoorn opened the pull request.

boutinb commented on inst/qml/Descriptives.qml:
```diff
@@ -405,61 +409,54 @@ Form
+				AvailableVariablesList
+				{
+					name: "heatmapVariables"
+					label: qsTr("")
```
no qsTr function with empty string

boutinb commented on inst/qml/Descriptives.qml:
```diff
@@ -382,17 +382,21 @@ Form
 				{ 
 					name: 				"densityPlotVariables"
-					label: 				qsTr("Density plots")
+					label: 				qsTr("")
```
no qsTr function with empty string

JohnnyDoorn commented:
@boutinb @vandenman 
I removed those empty qstr's, and added an option to the custom density plots, such that histograms are also available there. I also tweaked the transparency option to also only show when it actually has a use.
